onnxruntime/python/tools/transformers/fusion_conformer_attention.py

```diff
@@ -122,8 +122,8 @@ def fuse(self, normalize_node, input_name_to_nodes, output_name_to_node):
         if k_nodes is None:
             k_nodes = self.model.match_parent_path(
                 matmul_qk,
-                ["Transpose", "Transpose", "Reshape", "Add", "MatMul"],
-                [1, 0, 0, 0, 0],
+                ["Transpose", "Reshape", "Add", "MatMul"],
```

Comment:
Will this change break existing fusion that added in https://github.com/microsoft/onnxruntime/pull/23528?

Is the pattern caused by PyTorch or transformers package version? If no, please update related requirements.txt or document.